app/src/main/java/com/sun_asterisk/youtubebackground/screen/home/HomeAdapter.java

```diff
@@ -51,19 +57,30 @@ public int getItemCount() {
     class ViewHolder extends RecyclerView.ViewHolder {
         ImageView mImageVideo;
         TextView mTextTitle;
-        TextView mtextDescription;
+        TextView mTextDescription;
 
         ViewHolder(@NonNull View itemView) {
             super(itemView);
             mImageVideo = itemView.findViewById(R.id.imageVideo);
             mTextTitle = itemView.findViewById(R.id.textTitle);
-            mtextDescription = itemView.findViewById(R.id.textDescription);
+            mTextDescription = itemView.findViewById(R.id.textDescription);
+            itemView.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    if (mOnItemClickListener != null){
+                        int position = getAdapterPosition();
+                        if (position != RecyclerView.NO_POSITION){
+                            mOnItemClickListener.onItemClick(position);
```

Comment:
nếu code không quá dài và biến chỉ được sử dụng 1 lần thì e nên gán trực tiếp vào nhé
```suggestion
                            mOnItemClickListener.onItemClick(getAdapterPosition());
```